tensorbay/cli/utility.py

```diff
@@ -181,3 +181,14 @@ def _clean_up(editor_input: Optional[str]) -> Tuple[str, str]:
     if not cleaned_up_lines:
         return "", ""
     return cleaned_up_lines[0].lstrip(), "\n".join(cleaned_up_lines[1:])
+
+
+def error(message: str) -> NoReturn:
+    """Encapsulate click.echo() and sys.exit() when an error occurs.
```

Comment:
I can know this function is `echo` then `exit` from the source code easily. There is no need to describe this in docstring.